src/back-end/src/email-helpers.js

```diff
@@ -1,3 +1,5 @@
+// This file contains helpers used when we send custom emails to users.
```

Comment:
I mentioned to Aleks that we should include a slight [limitation](https://nodemailer.com/usage/using-gmail/#:~:text=Gmail%20has%20a%20limit%20of,recipients%20a%20day%20for%20free.) of nodemailer in our final documentation. Do you think it would be beneficial to note here that nodemailer (when used with Gmail) has a limit of 500 emails that can be sent per day as well?

Reply:
If it's in the limitations, I think it's okay not to add it here. Let's add it in the docs, not in the code.

Reply:
Sounds good!